2-structured-data/src/app.php

```diff
@@ -40,6 +43,35 @@
 
 $app['config'] = Yaml::parse(file_get_contents($config));
 
+// determine the datamodel backend using the app configuration
+$app['bookshelf.model'] = function ($app) {
+    /** @var array $config */
+    $config = $app['config'];
+
+    // Data Model
+    switch ($config['bookshelf_backend']) {
```

Comment:
TBH I really don't want to surround this (already verbose) datamodel switcher with an `isset`. If we have it set in the configs, and someone makes a typo and removes it, the warning (an exception) they get will be enough to help them debug.